settings/src/androidMain/kotlin/com/islamversity/settings/LicensesView.kt

```diff
@@ -0,0 +1,69 @@
+package com.islamversity.settings
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.islamversity.core.Mapper
+import com.islamversity.base.CoroutineView
+import com.islamversity.core.mvi.MviPresenter
+import com.islamversity.daggercore.CoreComponent
+import com.islamversity.settings.databinding.ViewLicensesBinding
+import com.islamversity.settings.di.DaggerLicensesComponent
+import com.islamversity.view_component.*
+import com.islamversity.settings.licenses.LicensesIntent
+import com.islamversity.settings.licenses.LicensesState
+import com.islamversity.settings.models.LicensesUIModel
+import com.islamversity.view_component.licenseItem
+import kotlinx.coroutines.channels.BroadcastChannel
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.merge
+import javax.inject.Inject
+import android.content.Intent
+
+
+class LicensesView : CoroutineView<ViewLicensesBinding, LicensesState, LicensesIntent>() {
+    @Inject
+    override lateinit var presenter: MviPresenter<LicensesIntent, LicensesState>
+    @Inject
+    lateinit var licenseMapper : Mapper<LicensesUIModel, LicenseItemModel>
+    private val intentChannel = BroadcastChannel<LicensesIntent>(Channel.BUFFERED)
```

Comment:
if you need this please consider using `CoroutineView.intents`
it's for the same purpose

Reply:
Can you please explain more?
I couldn't find anything related

Reply:
you did not use intentChannel anywhere so I said 
please remove this variable.

plus if you need a channel like this please don't create the variable yourself there is already one in the parent class:
`intents`